src/Notepads.Controls/Notepads.Controls.csproj

```diff
@@ -235,7 +235,7 @@
       <Version>2.0.6</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform">
-      <Version>6.2.10</Version>
+      <Version>6.2.11-preview-28914-04</Version>
```

Comment:
Should this be changed?

Reply:
Yeah, revert it and I can handle it offline.